snapcraft/internal/elf.py

```diff
@@ -77,7 +77,6 @@ def __init__(self, *, soname: str, path: str, root_path: str,
             self.in_base_snap = False
 
 
-@lru_cache()
```

Comment:
I'm curious, why did you remove it?

Reply:
simple pimple; on running `load_dependencies` to get the new ones, we'd get the old ones. There is no API to wipe the cache so I just decided to remove it